srcpkgs/ergo/patches/paths.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/default.yaml b/default.yaml
+index 36f3ee1e..51031fea 100644
+--- a/default.yaml
++++ b/default.yaml
+@@ -52,8 +52,8 @@ server:
+             # this is a standard TLS configuration with a single certificate;
+             # see the manual for instructions on how to configure SNI
+             tls:
+-                cert: fullchain.pem
+-                key: privkey.pem
++                cert: /var/lib/ergo/fullchain.pem
++                key: /var/lib/ergo/privkey.pem
```

Comment:
Isn't this going to result in a broken default configuration? There are no certificates installed in `/var/lib/ergo`. If so, I recommend dropping this patch entirely and shipping this as an example only. Alternatively, modify the configuration to disable TLS in the default, and let users configure it.

Reply:
I'm not married to this patch, but I added it more as a suggestion where to put the files, since the defaults are in the current directory which doesn't make much sense either

Reply:
We expect package users to understand how to use the software, and if the configuration is going to be non-functional anyway, I'd just drop the patch, `vsconf` the original as an example, and let people deal with it as they need it.